packages/sitecore-jss-dev-tools/src/codegen/extract-components.ts

```diff
@@ -0,0 +1,76 @@
+import { ExtractedFileType, resolveComponentImportFiles, sendCode } from './utils';
+import { fetchBearerToken } from '../auth/fetch-bearer-token';
+import chalk from 'chalk';
+import fs from 'fs';
+import path from 'path';
+
+type ExtractComponentArgs = {
```

Comment:
Just to allign with comments below
```suggestion
type ExtractComponentOptions = {
```